build.toml

```diff
@@ -26,3 +26,8 @@ name = 'Engine Libraries'
 type = 'lvBuildSpecAllTargets'
 project = '{cd}'
 build_spec = 'Engine Release'
+
+[package]
+type = 'nipkg'
+payload_dir = 'Built'
+install_destination = 'documents\\National Instruments\\NI VeriStand {veristand_version}\\SLSC Plugins\\Modules'
```

Comment:
This is a problem for older version of the custom device. For 2016 and earlier we need to install the package to Documents\National Instruments\NI VeriStand {veristand_version}\Custom Devices\SLSC Plug-ins\

From the [forums](https://forums.ni.com/t5/NI-VeriStand-Add-Ons-Documents/SLSC-Electronic-Datasheet-EDS-Plugin-Custom-Device/ta-p/3812754):

```
For VeriStand 2015 SP1:
C:\Users\Public\Documents\National Instruments\NI VeriStand 2015\Custom Devices\SLSC Plug-ins

For VeriStand 2016:
C:\Users\Public\Documents\National Instruments\NI VeriStand 2016\Custom Devices\SLSC Plug-ins

For VeriStand 2017:
 C:\Users\Public\Documents\National Instruments\NI VeriStand 2017\SLSC Plugins\Modules
```

Reply:
I know, but we also don't support the custom device earlier than 2018. I will begrudgingly allow different destinations based on build version, but I am wary of making changes to enable things that aren't officially supported in the master branch.

For instance, it is impossible to build the Scan Engine using what's in master for versions prior to 2017 due to API-breaking changes in the EtherCAT driver. I have no intention of making Scan Engine master buildable for older versions. Someone would need to create a legacy branch to fix up the files.